.code-samples.meilisearch.yaml

```diff
@@ -455,17 +456,14 @@ getting_started_add_documents_md: |-
   $movies = json_decode($movies_json);
 
   $client->index('movies')->addDocuments($movies);
-  ```
 
-  [About this SDK](https://github.com/meilisearch/meilisearch-php/)
+  /**
+   * More about this SDK: https://github.com/meilisearch/meilisearch-php
+   */
```

Comment:
I think we can drop the SDK links, feels a bit out of place. What do you think?

```suggestion
```

Reply:
I agree!